todo/src/actions/index.js

```diff
@@ -0,0 +1,21 @@
+export const MARK_DONE = 'MARK_DONE';
+export const ADD_TODO = 'ADD_TODO';
+
+export const markDone = (id) => (
+    { 
+        type: MARK_DONE,
```

Comment:
These actions look good Luis. You've got the pattern of importing your types down as well.